Add Zowe Explorer vNext (#177)
zowe/zowe.github.io: download.md

              </a>
            </div>
         </div>
      </div>
    </div>
  </div>
</div>
<div class="card-deck">
  <div class="card bg-light border-light mb-3">
  <h4 class="card-header">Zowe Explorer</h4>
    <div class="card-body">

>     <p class="card-text">The Zowe Explorer vNext release enables you to adopt the Team Configuration File (recently developed by the Zowe CLI Squad) - designed to make profile management more centralized and much simpler.</p>

Reviewer: ```suggestion
    <p class="card-text">Zowe Explorer vNext enables you to adopt the team configuration file, recently developed by the Zowe CLI Squad, to make profile management more centralized and much simpler.</p>
```

>     <p class="card-text">You can download the <em>.vsix</em> files directly via the GitHub release page with the tag <b>"Pre-release"</b>. Separate downloads exist for Zowe Explorer and Zowe Explorer Extension for FTP.</p>

Reviewer: ```suggestion
    <p class="card-text">You can download the <em>.vsix</em> files directly from the GitHub release page with the tag <b>"Pre-release"</b>. Separate downloads are available for Zowe Explorer and Zowe Explorer Extension for FTP.</p>
```
